ethpm_types/base.py

```diff
@@ -33,3 +37,21 @@ def json(self, *args, **kwargs) -> str:
             kwargs["exclude_none"] = True
 
         return super().json(*args, **kwargs)
+
+    @classmethod
+    @no_type_check
+    def _get_value(
```

Comment:
This won't work across the board unless we subclass this class then?

Reply:
Correct, every model that would need to validate Hexbytes would need to either use the basemodel from this library or add this method to their pydantic basemodel. The other piece is to use our specific version of HexBytes with the validation methods added.